CMake/sitkUseGTest.cmake

```diff
@@ -33,7 +33,9 @@ function(_sitk_gtest_use_gtest_source)
 
   set(BUILD_GTEST                 ON )
   set(BUILD_GMOCK                 OFF)
-
+  if ("${WIN32}")
+      set(gtest_disable_pthreads      ON CACHE INTERNAL "Disable use of pthreads in gtest")
+  endif()
```

Comment:
This change has already been integrated into the release branch. Please rebase your topic onto the current master, and do a force push.

```
git checkout RWindowsBattlesSquash
git fetch origin
git rebase origin/master
git push -f github_richardbeare HEAD
```